packages/@okta/vuepress-theme-prose/util/api.service.js

```diff
@@ -16,15 +17,18 @@ export class Api {
         ...this.baseheaders,
         ...options.headers
       },
-      data: options.body,
-    })
+      data: options.body
+    });
   }
 
   get(url, options = {}) {
-    return this.httpRequest('GET', url, options);
+    return this.httpRequest("GET", url, options);
   }
 
   post(url, options = {}) {
-    return this.httpRequest('POST', url, options);
+    options.headers = options.headers
+      ? { ...options.headers, ...this.postHeaders }
```

Comment:
FYI I think you only need this line:
```js
options.headers = {
  ...this.postHeaders,
  ...options.headers,
}
```

If `options.headers` is `undefined` it still is valid and works as expected (you can test in the Node CLI). Also note that `...options.headers` IMO should be second so it can override the default `postHeaders`.